### ISSUE TITLE
arm64 support for dimension integration manager

### ISSUE THREAD

**Issue by aurabindo** [spantaleev/matrix-docker-ansible-deploy#1210]
I just installed matrix on my odroid hc4, but the dimensions integration service failed, and systemctl status says:
```
Jul 29 16:20:23 odroid matrix-dimension[39340]: WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

**Comment by spantaleev** [comment]
The upstream image does not support ARM and we don't support [self-building](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/self-building.md) for that architecture yet.

You can try working on adding self-building support yourself (if it's even possible to build the Dimension image on ARM - I have some recollection that it's not) or contacting the upstream project to request pre-built ARM images.

**Comment by mattcen** [comment]
For the record, I've sent a PR to enable building on ARM, but haven't tweaked it to be acceptable yet: https://github.com/turt2live/matrix-dimension/pull/403

**Comment by johndotpub** [comment]
@aurabindo Feel free to check out the branch my recent PR is using and let shout if you have any issues with the self-build option for matrix-dimension.

**Comment by spantaleev** [comment]
Closed by #1445
